### PR TITLE
Support binary files in a mockup

### PR DESCRIPTION
**Pull request by MetroWind**
This commit adds support for binary files in a mockup. This is useful if a mockup contains things like a sysdump.

### CODE REVIEW

**Comment by MetroWind** [comment]
DCO check failed. I'll fix it and have another pull request.

File: redfishMockupServer.py [hunk 1]
```diff
@@ -515,20 +515,41 @@ def do_GET(self):
...
-                    f = open(fpath_xml, "r")
-                elif os.path.isfile(fpath_direct):
-                    filename, file_extension = os.path.splitext(fpath_direct)
-                    file_extension = file_extension.strip('.')
-                    f = open(fpath_direct, "r")
+            elif os.path.isfile(fpath_xml):
+                f = open(fpath_xml, "r")
                 self.send_response(200)
-                self.send_header("Content-Type", "application/" + file_extension + ";odata.metadata=minimal;charset=utf-8")
+                self.send_header("Content-Type", "application/xml;odata.metadata=minimal;charset=utf-8")
```
Comment (mraineri) [inline redfishMockupServer.py]: Can we remove the "odata.metadata=minimal" from here and below while we're at it? We've been moving away from these tags on content-type

Comment (MetroWind) [reply redfishMockupServer.py]: Done.

**Comment by mraineri** [comment]
Approved 1/20

**Pull request merged**